### ISSUE TITLE
AoT compilation fails

### ISSUE THREAD

**Issue by Rob-Post** [rintoj/angular2-virtual-scroll#16]
When running AoT compilation ( node_modules/.bin/ngc -p tsconfig-aot.json ) it fails with the following error:

`Unexpected value 'VirtualScrollModule in /node_modules/angular2-virtual-scroll/dist/virtual-scroll.d.ts' imported by the module 'AppModule in /app/app.module.ts'`

tsconfig-aot.json
```
{
  "compilerOptions": {
    "module": "es2015",
    "moduleResolution": "node",
    "target": "es5",
    "noImplicitAny": false,
    "sourceMap": true,
    "mapRoot": "",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "lib": [
      "es2015",
      "dom"
    ],
    "outDir": "lib",
    "skipLibCheck": true,
    "rootDir": ".",
	"typeRoots": [
      "node_modules/@types"
    ]
  },
  "angularCompilerOptions": {
    "genDir": "./app/ngfactory",
    "entryModule": "./app/app.module#AppModule"
  }
}
```

app.module.ts (excerpt)

```
import { VirtualScrollModule } from 'angular2-virtual-scroll';

@NgModule({
  imports: [
	VirtualScrollModule,
  ],
  declarations: [

  ],
  bootstrap: [ AppComponent ]
})
```

**Comment by jsen-** [comment]
Seems like AOT does not like the distributed `.js` file, but requires the original `.ts` file.
This seems to work...
```javascript
// FIXME: quick-hack to fix AOT
import { VirtualScrollModule } from "../../resources/angular2-virtual-scroll/src/virtual-scroll"
```

**Comment by ronzeidman** [comment]
Notice that if you provide the metadata.json files alongside the .js files (meaning you need to AOT compile this library in advance) the .ts files are no longer needed.
https://medium.com/@isaacplmann/getting-your-angular-2-library-ready-for-aot-90d1347bcad#.wxi37ud13

**Comment by rintoj** [comment]
Fixed with v0.1.0. Now this library is AoT enabled.
